src/components/TeacherViews/LessonOverview/LessonOverview.view.tsx

```diff
@@ -226,6 +226,7 @@ export function LessonOverview({ lesson }: LessonOverviewProps) {
           keyLearningPoints,
         )}
         showDownloadAll={showDownloadAll}
+        isEYFS={keyStageSlug === "early-years-foundation-stage"}
```

Comment:
think it would be a bit cleaner if we have a `showShare` prop instead and pass that through to the button component with an early null return like we do with `showDownloadAll`